lib/cli/index.js

```diff
@@ -32,6 +32,7 @@ exports.run = () => {
         .allowUnknownOption(true)
         .option('-r, --reporter <reporter>', 'test result reporter (flat by default)', collect)
         .option('--html-reporter-path <path>', 'relative path where html reporters will be stored', collect)
+        .option('--html-failed-only', 'html report generates only for failed tests')
```

Comment:
Okay, I agree with you. If it will be really necessary then we will implement it.